app/pages/nonsipp/membersurrenderedbenefits/SurrenderedBenefitsAmountPage.scala

```diff
@@ -17,16 +17,32 @@
 package pages.nonsipp.membersurrenderedbenefits
 
 import utils.RefinedUtils.RefinedIntOps
+import utils.PageUtils.removePages
+import queries.Removable
 import models.SchemeId.Srn
 import play.api.libs.json.JsPath
-import models.Money
+import models.{Money, UserAnswers}
 import config.Refined.Max300
 import pages.QuestionPage
 
+import scala.util.Try
+
 case class SurrenderedBenefitsAmountPage(srn: Srn, memberIndex: Max300) extends QuestionPage[Money] {
 
   override def path: JsPath = Paths.memberPensionSurrender \ toString \ memberIndex.arrayIndex.toString
 
   override def toString: String = "totalSurrendered"
 
+  override def cleanup(value: Option[Money], userAnswers: UserAnswers): Try[UserAnswers] =
```

Comment:
Is there a chance that this cleanup breaks the removal through `RemoveSurrenderedBenefitsController`? (Same question applies for all member payment cleanup methods)